frame/elections-phragmen/src/lib.rs

```diff
@@ -653,9 +657,13 @@ decl_module! {
 
 impl<T: Config> Module<T> {
 	/// The deposit value of `count` votes.
-	fn deposit_of(count: usize) -> BalanceOf<T> {
-		T::VotingBondBase::get().saturating_add(
-			T::VotingBondFactor::get().saturating_mul((count as u32).into())
+	fn deposit_of(count: u32) -> BalanceOf<T> {
+		T::VotingBondStorageBase::get().saturating_add(
+			T::VotingBondStorageFactor::get().saturating_mul(count.into())
+		).saturating_add(
+			T::VotingBondWeightFactor::get().saturating_mul(
+				T::WeightInfo::election_phragmen(0, 1, count).saturated_into()
```

Comment:
note that this was wrong before: additional weight to reserve is the weight for 0 candidate, 1 additional voter, and `count` votes (edges).

Reply:
ah, thanks!